xwiki-platform-core/xwiki-platform-icon/xwiki-platform-icon-macro/src/main/java/org/xwiki/icon/macro/DisplayIconMacroParameters.java

```diff
@@ -100,4 +102,24 @@ public void setFallback(boolean fallback)
     {
         this.fallback = fallback;
     }
+
+    /**
+     * @since 16.5.0RC1
+     * @return the text alternative picked for the icon
+     */
+    public String getTextAlternative()
+    {
+        return this.textAlternative;
+    }
+
+    /**
+     * @since 16.5.0RC1
```

Comment:
Here too

Reply:
Addressed in https://github.com/xwiki/xwiki-platform/pull/3123/commits/2f71bec475271782aa425310b97eceacc021accb :+1: